browser.js

```diff
@@ -16,7 +16,7 @@ var Bootstrapper = require('./lib/bootstrapper').Bootstrapper;
  * @header boot(app)
  */
 
-exports = module.exports = function bootBrowserApp(app, options) {
+exports = module.exports = function bootBrowserApp(app, options, callback) {
```

Comment:
Since this is for LoopBack Next, can we drop support for callbacks and switch exclusively to Promise-based APIs here in loopback-boot?